zebedee-cms/src/main/java/com/github/onsdigital/zebedee/service/ServiceStoreImpl.java

```diff
@@ -1,60 +1,79 @@
 package com.github.onsdigital.zebedee.service;
 
-import com.github.onsdigital.zebedee.model.PathUtils;
 import com.github.onsdigital.zebedee.model.ServiceAccount;
 import com.github.onsdigital.zebedee.util.serialiser.JSONSerialiser;
-import org.apache.commons.lang3.StringUtils;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.nio.file.FileAlreadyExistsException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 
+import static com.github.onsdigital.zebedee.logging.CMSLogEvent.error;
+
 public class ServiceStoreImpl implements ServiceStore {
 
-    private final Path rootLocation;
+    private final Path serviceRoot;
 
     private final JSONSerialiser<ServiceAccount> jsonSerialise;
 
     private final static String JSON_EXTENSION = ".json";
 
-    public ServiceStoreImpl(Path rootLocation) {
-        this.rootLocation = rootLocation;
+    public ServiceStoreImpl(Path serviceRoot) {
+        this.serviceRoot = serviceRoot;
         this.jsonSerialise = new JSONSerialiser<>(ServiceAccount.class);
     }
 
+    public ServiceStoreImpl(Path serviceRoot, JSONSerialiser<ServiceAccount> jsonSerialise) {
+        this.serviceRoot = serviceRoot;
+        this.jsonSerialise = jsonSerialise;
+    }
+
     @Override
-    public ServiceAccount get(String id) throws IOException {
-        ServiceAccount service = null;
-        final Path path = getPath(id);
-        if (Files.exists(getPath(id))) {
-            try (InputStream input = Files.newInputStream(path)) {
-                service = jsonSerialise.deserialiseQuietly(input, path);
-            }
+    public ServiceAccount get(String token) throws IOException {
+        Path filePath = null;
+        ServiceAccount account = null;
+
+        if (ServiceTokenUtils.isValidServiceToken(token)) {
```

Comment:
will we still have bearer prefix here?

Reply:
No. The prefix should be removed by this point.